src/neptune/internal/operation_processors/operation_storage.py

```diff
@@ -34,29 +34,46 @@ def __init__(self, data_path: Union[str, Path]):
             self._data_path = data_path
         else:
             self._data_path = Path(data_path)
-
         # initialize directories
         os.makedirs(self.data_path, exist_ok=True)
         os.makedirs(self.upload_path, exist_ok=True)
 
+        cwd = Path(os.getcwd())
+        self._data_path_abs = cwd / self._data_path
+
     @property
     def data_path(self) -> Path:
         return self._data_path
 
+    @property
+    def data_path_abs(self) -> Path:
+        return self._data_path_abs
+
     @property
     def upload_path(self) -> Path:
```

Comment:
There are only two usages of this in `operation.py`. It could be changed to an absolute path right here.